byoeb-v1/byoeb/tests/integration/conftest.py

```
@@ -0,0 +1,115 @@
+def auth_access_token(envs, auth_session):
+    token = auth_session.cookies.get("asha_auth_token")
+    if not token: raise RuntimeError("Auth cookie not set by /auth/token/issue.")
+    return token
+
+
+@pytest.fixture(scope="session")
+def auth_session(envs):
+    import time as _time
+    session = requests.Session()
+    headers = {"Content-Type": "application/x-www-form-urlencoded", "X-Tenant-ID": str(envs.auth_tenant_id)}
+    for attempt in range(6):
```

Ruff (0.15.10)

[warning] Loop control variable `attempt` not used within loop body

Rename unused `attempt` to `_attempt`

(B007)